README.md

```diff
@@ -20,21 +20,25 @@ Check [here](https://github.com/cylc/cylc-flow/issues/2752) for info on supporte
 
 Install from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=cylc.vscode-cylc).
 
+## Issues
+
+Please report any syntax highlighting issues at [cylc/cylc-textmate-grammar](https://github.com/cylc/cylc-textmate-grammar/issues). Any other issues with the extension should be reported [here](https://github.com/cylc/vscode-cylc/issues).
+
 ## Contributing
 
-This repo includes the [cylc/cylc-textmate-grammar](https://github.com/cylc/cylc-textmate-grammar) repo as a git submodule in the `/syntaxes` directory. If you don't have experience with submodules, you should [read the docs](https://git-scm.com/book/en/v2/Git-Tools-Submodules) first.
+This repo includes the [cylc/cylc-textmate-grammar](https://github.com/cylc/cylc-textmate-grammar) repo as a git submodule in the `/syntaxes/` directory. If you don't have experience with submodules, you should [read the docs](https://git-scm.com/book/en/v2/Git-Tools-Submodules) first.
 
 The cylc-textmate-grammar repo contains a JSON TextMate grammar file which is used by VSCode for syntax highlighting. Read the [VSCode syntax highlight guide](https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide) for more information. **Note:** do not edit the JSON file when contributing; instead you should edit the JavaScript grammar file and build it, as explained in the [contributing](https://github.com/cylc/cylc-textmate-grammar#contributing) section of cylc-texmate-grammar.
 
-To install a development version of this extension, first find your VSCode extensions folder (should normally be `~/.vscode/extensions/`). Then:
```

Comment:
Why is this bit no longer rqd?

Reply:
It seems this is not the intended way to develop extensions, instead you're meant to clone to a different location and run a debugging session using F5: https://github.com/microsoft/vscode/issues/25159#issuecomment-296709441

Reply:
Doesn't surprise me - what I was doing was quick and dirty.